### ISSUE TITLE
Error during the project generation

### ISSUE THREAD

**Issue by zonetecde** [Alexiril/PonyKey#1]
Hello,

After installing yaml using `pip install PyYAML` (it would be nice to specify this in the doc), the build succeeds but this error appears: 

![image](https://github.com/Alexiril/PonyKey/assets/56195432/7441bf03-be01-41dc-8323-5ba8f49d7070)

The problem is here, in the last line :

```cs
    private static dynamic GetMultipliedValue(JArray? values)
    {
        if (values == null) throw new NullReferenceException("Not correct values to multiply");
        return values.Select(value => CompileValue(value) ?? 1).Aggregate<dynamic, dynamic>(1f, (current, val) =>
            current * (float.TryParse(val.ToString(), out float floatValue) ? floatValue : val));
    }
```

In debug mode, `current` has a value of 1, `val` has a value of 1280 and `floatValue` has a value of 0

**Comment by Alexiril** [comment]
**Hello!**  
Thank you so much for your help. You were right, I've fixed the scene building system, and now it's ready to go, check it out!

By the way, speaking of python and the pyYaml module, I have mentioned it in the README.md, after the last refactoring you don't need to use any third-party modules, now python uses only built-in modules.

Thanks again, and please, check again if the problem still occurs on your PC!

**Comment by zonetecde** [comment]
Thank you, and sorry for the late answer.

There's some problem I faced when compiling the project:  

**The first was this error:**

`The ...\Engine\.config\dotnet-tools.json file came from another computer and was possibly blocked to protect that computer.`

To fix this error I had to create the same file with the same name and same container myself and put it in the same folder

**The second error is :**

`error CS0006: Metadata file '...\Engine\obj\Debug\net6.0-windowsref\Engine.dll' not found
`
You should create a folder in the visual studio project and set the dll file to 'copy to output directory'.

**Comment by Alexiril** [comment]
**Hello again :)**
Yeah, you are 100% right about the first error, but I found the way out. Now the file generates before the build starts, so it will never be from another computer.

Speaking of the second error, it is really weird. As if it works how it should, but actually, there's an odd part in the error message:
`error CS0006: Metadata file '...\Engine\obj\Debug\net6.0-windows` and there should be another slash here `ref\Engine.dll' not found `
When built correctly, the file `Engine\obj\Debug\net6.0-windows\ref\Engine.dll` generates, but something really peculiar had happened in your build, there is no slash in the path.

![image](https://github.com/Alexiril/PonyKey/assets/96677511/fecffefe-2c26-4f56-9cb3-c59f8a2ddbcf)

**I will try to find out how to recreate the error and how to fix it.**

**Comment by zonetecde** [comment]
It seems that solving the first problem also solved the second! I can finally build the project without errors, but now there's another error ^^'

![image](https://github.com/Alexiril/PonyKey/assets/56195432/ae6f83d7-d541-4ec6-97a9-891ef046c602)

![image](https://github.com/Alexiril/PonyKey/assets/56195432/7f02a92f-1b79-4cfd-bc5d-305c64a490b8)

![image](https://github.com/Alexiril/PonyKey/assets/56195432/95ed8645-07d5-4bb7-9f9a-88960d48f346)

_note: even after commenting on these lines other future errors appear, why not make an installer for the game? :p_

**Comment by Alexiril** [comment]
I've fixed the float error, just I'm new to different culture decimals, didn't think about that before :)

As well, I've posted the built release of the current game, so now it's not necessary anymore to build it from source. But I want to warn that this game is not really ready, I'm working right now on the interesting continuation and don't post it to the GitHub yet. (it's coded badly, due to Monogame, I want to try to get rid of Monogame dependencies). So, in the release there's only the ready part, not so interesting :)

I hope now you finally can check the game out :smile: Anyway, feel free to continue the issue if something doesn't work out.
